app/helpers/application_helper.rb

```diff
@@ -1,4 +1,13 @@
 # frozen_string_literal: true
 # General view helper module
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
```

Comment:
This should have YARD-style documentation - take a look throughout the codebase for examples.